src/features/soft_navigations/constants.js

```diff
@@ -0,0 +1,29 @@
+import { FEATURE_NAMES } from '../../loaders/features/features'
+
+export const INTERACTION_TRIGGERS = [
+  'click', // e.g. user clicks link or the page back/forward buttons
+  'keydown', // e.g. user presses left and right arrow key to switch between displayed photo gallery
+  'submit' // e.g. user clicks submit butotn or presses enter while editing a form field
+]
```

Comment:
We should probably evaluate if this covers the cases we want (including touch events, scroll events, etc)

Reply:
AFAICF(ind), these are all that Google said they're looking at. This area could use feedback from trial users on, me thinks.